docs/operations/versioning.md

```diff
@@ -0,0 +1,305 @@
+
+- Adding `preview` versions
+- Promotion of a `preview` version to `supported`. We think this should be done on a case-by-case basis, hence Gardener deliberately does not offer an automation in this case.
+- Deprecate a `supported` version
+   - This is of particular importance when using `Version Management`. As it defines an initial classification for versions with no classification, 
+ it can happen that it tries to introduce another `supported` version even though there is already one present. That is forbidden and blocked by the Gardener API server. 
+ That situation occurs when specifying an unclassified version that is the overall latest version (or the latest version of a minor version when using `Version Maintenance`).
+ It is the operator duty to make sure that this does not happen. Specifically when introducing a new latest version without a classification, the current `supported` version should be manually deprecated ahead of time.
+- Deletion of versions. Before deletion, the operator has to make sure the version is not in use by any Shoot. The recommended way to achieve this, is to deprecate the version and set an expiration date. 
+Clusters with `auto-update` configured will automatically update during their next maintenance. The remainder is forcefully updated during maintenance time after the version is finally expired.
+- Setting an `expiration date` on the latest patch version of an `unmaintained`  Kubernetes minor version. This is to prevent that Gardener automatically forces minor version upgrades for Clusters.
+    - After the deprecation date passed, it will force a Kubernetes minor version update during the next maintenance time window of the Shoot.
+
+## Common scenarios for the Gardener Operator
+
+The following describes common scenarios the Gardener operator regularly faces when managing the Kubernetes and Machine image versions of a Gardener installation.
+We assume a CloudProfile that looks schematically like this:
+
+``` yaml
+1.18.4 :  preview
+1.18.3 : (VersionMaintenance: supported, otherwise: deprecated)
+1.17.0 : (VersionMaintenance: supported, otherwise: deprecated)
+1.16.0 : (VersionMaintenance: supported, otherwise: deprecated)
+1.15.1 : deprecated with no expiration date
+1.15.0 : deprecated with expiration date that is not expired
+1.14.0 : deprecated with expiration date already expired
+```
+For `Version Maintenance`, the Gardener Controller Manager configuration in `controllers.cloudProfile.kubernetesVersionManagement.versionMaintenance` is set to `3`.
+
+**Scenarios**
+
+* * *
+***A new patch version has been released and should be added to the CloudProfile.***
+
+**Example**: Kubernetes releases version 1.18.4. 
+ 
+ -  Add the version and set the classification to `preview`. There can be multiple preview versions.
+ 
+**Example 2**: Kubernetes releases version 1.17.1. 
+ 
+ - With `Version Management` enabled: Version 1.17.1 will be deprecated wit an expiration date right away.
```

Comment:
```suggestion
 - With `Version Management` enabled: Version 1.17.1 will be deprecated with an expiration date right away.
```

Reply:
Why will it be deprecated right away? Do you mean 1.17.0?

Reply:
No, using VersionManagement enforces only one supported version overall. The supported version in this example is already 1.18.3 - so the version 1.17.1 will be deprecated accordingly.